src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/DeveloperCredentials/MsalTokenCredential.cs

```diff
@@ -111,22 +119,26 @@ public override async ValueTask<AccessToken> GetTokenAsync(TokenRequestContext r
             {
                 if (ex.Message.Contains("AADSTS70002")) // "The client does not exist or is not enabled for consumers"
                 {
-                    Console.WriteLine("An Azure AD tenant, and a user in that tenant, " +
-                        "needs to be created for this account before an application can be created. See https://aka.ms/ms-identity-app/create-a-tenant. ");
+                    LogError("An Azure AD tenant, and a user in that tenant, " +
+                        "needs to be created for this account before an application can be created. " +
+                        "See https://aka.ms/ms-identity-app/create-a-tenant. ");
                     Environment.Exit(1); // we want to exit here because this is probably an MSA without an AAD tenant.
                 }
 
-                Console.WriteLine("Error encountered with sign-in. See error message for details:\n{0} ",
-                    ex.Message);
+                LogError("Error encountered with sign-in. See error message for details:\n{ex.Message}");
                 Environment.Exit(1); // we want to exit here. Re-sign in will not resolve the issue.
             }
             catch (Exception ex)
             {
-                Console.WriteLine("Error encountered with sign-in. See error message for details:\n{0} ",
-                    ex.Message);
+                LogError($"Error encountered with sign-in. See error message for details:\n{ex.Message}");
                 Environment.Exit(1);
             }
             return new AccessToken(result.AccessToken, result.ExpiresOn);
         }
+
```

Comment:
Why not make `LogError` an extension on IConsoleLogger and remove this private method? I imagine this could be used in other places (we can wait to convert all of them if that helps)